conman/redirects/models.py

```diff
@@ -3,6 +3,7 @@
 from django.utils.translation import ugettext_lazy as _
 
 from conman.routes.models import Route
+
```

Comment:
I've played a lot with the isort settings, but can't avoid this particular bit of whitespace. I can put up with it, but if there's a solution, I'd love to hear it.